src/server/controller/assessment/index.ts

```diff
@@ -1,15 +1,21 @@
+import { AssessmentFileRepository } from '@server/repository/assessment/file'
 import { TableRepository } from '@server/repository/assessment/table'
 
 import { create } from './create'
 import { createCycle } from './createCycle'
+import { createFile } from './createFile'
 import { getOne, getOneWithCycle } from './getOne'
 import { remove } from './remove'
 
 export const AssessmentController = {
   create,
+  createCycle,
+  createFile,
+  getFile: AssessmentFileRepository.getOne,
+  getFiles: AssessmentFileRepository.getMany,
   getOne,
   getOneWithCycle,
-  remove,
-  createCycle,
   getTable: TableRepository.getOne,
+  remove,
+  removeFile: AssessmentFileRepository.remove,
```

Comment:
File related methods should be moved to file controller

Reply:
file controller doesn't exist. do I have to create one and put there getAssessmentFile, getAssessmentFiles and removeAssessmentFile?